### PR TITLE
Editorial Edits

### PR DESCRIPTION
**Pull request by DukeofSuffolk**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -7,235 +7,260 @@
 Abstract
 ========
 
-This technote details the types of documentation that are available to SIT-Com personnel, and when they should be used.
+This technote details the types of documentation that are available to System Integration Test & Commissioning (SIT-COM) personnel, and when they should be used.
```
Comment (patrickingraham) [inline index.rst]: It should be SIT-Com not SIT-COM.... everywhere...

File: index.rst [hunk 2]
```diff
@@ -7,235 +7,260 @@
...
+This includes any Jupyter notebooks and Jira tickets where work was performed, especially if there exists content in the external source which may be applicable to the technote.
+Inside the technote's repository is a ``_static`` directory where other files (i.e., static objects) can be included, such as any analysis Jupyter Notebook referenced by the text or supporting figures.
 
-Technotes should *never be deleted*.
-In the event that the content becomes stale, superseded, or even found to be incorrect, a new version of the technote should be committed that removes the content and has ``DEPRECATED`` in the title.
-If applicable, the replacement document(s) should be linked.
-This way the content is never entirely lost as it will remain in the git history of the repository.
+**Technotes should never be deleted**.
+In the event that the content becomes stale, superseded, or even found to be incorrect, a revision to the technote should be committed removing or correcting the content.
+As appropriate, included ``DEPRECATED`` in the title and link any documents that replace the technote.
```
Comment (patrickingraham) [inline index.rst]: include (not included)

File: index.rst [hunk 3]
```diff
@@ -7,235 +7,260 @@
...
 
-   Notebooks are not to hold functional code over extended periods of time (~2 weeks) nor are they meant to augment observatory control and/or reduction software. 
-   If a piece of code (e.g. a function) developed in a notebook and is useful then it must be moved into a function in the development repositories.
-   In this case of control system code, this is further  discussed in `tstn-010 <https://tstn-010.lsst.io/>`_.
+   Notebooks are not to hold functional code over extended periods of time (~2 weeks), nor are they meant to augment observatory control and/or reduction software.
+
+   If a piece of code (e.g., a function) developed in a notebook is useful, then it must be moved into a function in the development repositories.
+   In the case of control system code, this workflow is discussed in `TSTN-010 <https://tstn-010.lsst.io/>`_.
+
+A easy indication is that if one finds themselves copying/pasting code from a notebook to another, then that code should not be in a notebook!
```
Comment (patrickingraham) [inline index.rst]: An easy...

File: index.rst [hunk 4]
```diff
@@ -7,235 +7,260 @@
...
+To store in this area requires a Zephyr Jira Test Case counterpart.
 Details on how to use this area is found in the README file of the repository.
 
 Review Criteria
 """""""""""""""
 
-This section will be populated in future versions.
-However, as mentioned previously, the expectation is that the notebooks closely resemble the examples found in the `AuxTel area <https://github.com/lsst-ts/ts_notebooks/tree/develop/procedures/auxtel>`_.
+Official artifacts for verification require specified review criteria.
+Standardizing this is underway, and this section will be populated in a future revision of this document.
```
Comment (patrickingraham) [inline index.rst]: I think it is standardized... I just don't know enough about it... and the people reading this document won't be using it as a reference for doing verification docs... 
Consider:
This section will be populated in a future revision of this document by the verification coordinators.

File: index.rst [hunk 5]
```diff
@@ -7,235 +7,260 @@
...
+
+.. note::
+
+   Adding a ``/v`` to a webspace URL will display the rendered versions of all branches in the repository.
 
 Webspaces are areas are best used for user-facing documentation.
-This includes general information, how-to's and procedures that are not subject to change control, and therefore do not risk the safety of personnel or equipment.
-Anything requiring strict reviews (e.g. glass lift plans) cannot be put into a webspace, but they can be linked!
+This includes general information, how-to documentation and procedures that are not subject to change control (therefore, these procedures are restricted to ones that do not risk the safety of personnel or equipment).
+Anything requiring strict reviews (e.g., glass lift plans) cannot be put into a webspace, but they can be linked within webspaces.
```
Comment (patrickingraham) [inline index.rst]: "from webpaces" no?
They will be hosted elsewhere (e.g docushare)

**Review by patrickingraham — APPROVED** [review]
Please see comments.
The "SIT-Com" one is probably the most significant whereas the others are minor.

File: index.rst (hunk 1 above)
Comment (DukeofSuffolk) [reply index.rst]: Reverted back in latest commit; matches capitalization typically seen in reviews.

File: index.rst (hunk 2 above)
Comment (DukeofSuffolk) [reply index.rst]: See change in latest commit.

File: index.rst (hunk 3 above)
Comment (DukeofSuffolk) [reply index.rst]: See change in latest commit.

File: index.rst (hunk 4 above)
Comment (DukeofSuffolk) [reply index.rst]: Changed in latest commit to the suggestion.

File: index.rst (hunk 5 above)
Comment (DukeofSuffolk) [reply index.rst]: Changed in latest commit to better indicate that this type of information is never in a webspace.

**Pull request merged**